examples/declarative/after.py

```diff
@@ -0,0 +1,143 @@
+import typing
+from dataclasses import dataclass
+
+from hiku.directives import Deprecated
+from hiku.expr.core import S
+from hiku.sources.graph import SubGraph
+
+import hiku.declarative.graph as hiku
+
+from .common import (
+    PARTNERS,
+    SPACESHIP_IDS,
+    DroidCtx,
+    get_droid_labels,
+    get_episodes,
+    get_full_name,
+    get_human_status_def,
+    low_level_graph,
+)
+
+
+human_sg: typing.Any = SubGraph(low_level_graph, "human")
+
+
+def link_partner(ids: list[int], ctx: dict) -> list[hiku.link["Human"]]:
+    return [hiku.link.from_(PARTNERS.get(i, 0)) for i in ids]
```

Comment:
Why do we need `.from_` ?

Reply:
I believe it had to do with a variance, but probably can remove now